cmd/mt-kafka-mdm-report-out-of-order/flags.go

```diff
@@ -0,0 +1,135 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/grafana/globalconf"
+	"github.com/grafana/metrictank/idx/cassandra"
+	inKafkaMdm "github.com/grafana/metrictank/input/kafkamdm"
+	log "github.com/sirupsen/logrus"
+)
+
+type Flags struct {
+	flagSet *flag.FlagSet
+
+	runDurationStr string
+	Config         string
+	PartitionFrom  int
+	PartitionTo    int
```

Comment:
I hadn't put this in the config because I didn't see any pattern around that in any of the metrictank CLIs (all options for the programs themselves are in flags). It would be simple enough to add though (just an additional globalconf register). What do you think? Either way you want/need to match what's in the kafka-mdm section since there isn't a way to pull config from a separate section and kafka-mdm does not expose this.

Reply:
@shanson7 I don't feel too strongly about this, if you think it makes more sense I can make this change.